### ISSUE TITLE
Thanks for the screenshot!  Really helped understand your program

### ISSUE THREAD

**Issue by PySimpleGUI** [kennyh7279/shortcut-generator-simple#1]
It's awesome to see a Linux shortcut generator.  I've got one for windows... psgshortcut.. but it's windows specific. I wasn't quite sure what the best approach would be for Linux users.

**Comment by kennyh7279** [comment]
Glad and shocked someone liked it/even noticed.  Yeah I got tired making .desktop files by hand for certain things so I just whipped this up.  Still no idea what i'm doing, still learning though.

**Comment by PySimpleGUI** [comment]
![image](https://user-images.githubusercontent.com/46163555/183310876-4c2f862d-acb3-4359-a4d7-ab801fdfc466.png)


I love it!  

Still no idea what you're doing (your words / your opinion), still learning, and willing to put your work up on GitHub for the world to see.  That's bravery right there... and bravery I notice, admire, and find much inspiration in.  Way to go Kenny!

Here's the shortcut project I did a few months ago:
https://github.com/PySimpleGUI/psgshortcut

I  too didn't know what I was doing.  And, I also got tired of manually doing things.  It's really great to see you build something to solve your problem and then share it with the rest of us.  

I'm really happy to see your shock is a happy one.  Keep building and sharing!  You're on a great path.
